omnibus/files/private-chef-upgrades/001/035_solr_to_elasticsearch_migration.rb

```diff
@@ -25,6 +25,9 @@
       log "Migrating indexed search data..."
       run_command("chef-server-ctl reindex -a -d -t -w")
 
+      # run the cleanup for post upgrade
+      run_command("chef-server-ctl cleanup")
```

Comment:
I don't think we want to do this.  This is run by the user once they are sure the upgrade has been successful.

Reply:
fair enough - I went back and forth on this, will remove and add to the docs instead.